### DISCUSSION TITLE
Adjust Y ticks value for grouping same answer for different group

### DISCUSSION THREAD

**Discussion by dhcvn** [nmalkin/plot-likert#46] [General]
Hi nmalkin, 
I have plotted same question for different groups. I would like to adjust yticks value so that all bars of the one question is close and has distant with other question. For example in the picture below:
![image](https://github.com/nmalkin/plot-likert/assets/40560482/0d4da197-6f08-40a5-b1eb-3d1b6799997f)
I want the bar of q1.1_r1 is next to the bar of q1.1_r2. 
Thank you in advance for your response!

**Comment by nmalkin** [comment]
Hi!

I want to make sure I understand your question correctly: are you trying to adjust the vertical space between the bars, such that there's no (or minimal) space between q1.1_r1 and q1.1_r2, then a big(ger) space, then again q1.2_r1+q1.2_r2, and so on?

If so, then unfortunately I don't have a good solution for you. I recently had to do it for one my graphs, and the only way I could figure out to do it was by abandoning the `pandas.plot` functions and manually drawing each bar using matplotlib, similarly to how it's done in this example: https://matplotlib.org/stable/gallery/lines_bars_and_markers/barchart.html

(If anyone knows of a better way, I'd love to hear it!)
